docs/advanced/application-gateway/cli/create-ag.sh

```diff
@@ -75,14 +75,14 @@ done
 
 # Remove it if the application gateway (AG) exists
 echo "Removing it if the application gateway exists"
-azure network application-gateway delete --resource-group $rgName --name $appgwName --quiet
+azure network application-gateway delete --resource-group $rgName --name $appgwName --quiet || true
```

Comment:
@niroyb 
If azure cli failed to delete the application gateway, the script will continue because of `|| true`.
It's better to check whether the application gateway exists before deleting it.

Reply:
After the changes, please rebase your codes with the latest commits.